tensorflow_model_optimization/g3doc/guide/clustering/clustering_with_keras.ipynb

```diff
@@ -0,0 +1,510 @@
+{
```

Comment:
<p>1. Would use the simpler model and preprocessing from <a href="https://www.tensorflow.org/model_optimization/guide/quantization/training_example" target="_blank">https://www.tensorflow.org/model_optimization/guide/quantization/training_example</a></p><p> </p><p>It'll train faster and to understand how clustering works, a complex model isn't necessary.</p><p> </p><p>2. It's not too useful to evaluate the model here. The main value of this baseline is to then compare with clustering, so we may as well have this logic closer to where it's useful. The same thing for saving the model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<ol><li>Yes, that is probably good. Will the pruning end-to-end example have the same model as the one in the trainnig_example as well? Should we keep the clustering example similar to the pruning one since they are similar and it's easier for the user to see their individual benefits or the combined one? </li><li>Right, OK.</li></ol><p> </p>

Reply:
> Will the pruning end-to-end example have the same model as the one in the trainnig_example as well?  

Yes, it'll use the same model as the quantization one. 

The pruning docs are in the midst of completely being revamped.